### ISSUE TITLE
Cannot auto attack with bow using right-click as a hunter

### ISSUE THREAD

**Issue by Bigmason123** [elysium-project/server#2058]
First of all, thank you for taking the time to report this issue.

Please note that the bug tracker does not accept reports on account or character
issues that are unrelated to the server's development. You should open a ticket
in game and wait for the response of a gamemaster. They may redirect you to this
bug tracker if they believe there's a server issue that they cannot help you
with.

Before writing your report, please do read and consider the following points to
increase the quality of your report.

- Search the issue tracker for open and closed issues before opening a new
  issue. You may be able to contribute additional facts by commenting on
  existing issues instead of adding extra noise in a separate issue.

- http://www.wowhead.com/ and http://db.vanillagaming.org/ are not valid sources
  for quests, items etc., instead, please use archived versions of
  http://wow.allakhazam.com or http://www.thottbot.com through
  https://archive.org/.

- Validate the date and time of the archived sources to a certain patch. The
  patch release dates can be found on the corresponding WoWWiki article, as
  found at http://wowwiki.wikia.com/wiki/Category:World_of_Warcraft_patches.

- Videos are valid if and only if the footage contains a timestamp or
  information that confirms the time it was recorded. Date of upload is NOT
  valid and does not confirm when it was shot.

- Read the Effective Bug Reporting article at
  https://github.com/elysium-project/server/blob/development/docs/CONTRIBUTING.md
  which covers these points and more.

Once you've read this message, delete it and write your report below.

--
Hello there!!

Let me preface this by saying how insanely excited I am to be able to play Vanilla WoW for the first time since 2006, this is very nostalgic for me and it means a great deal!
Now then, down to brass taxes...
I jsut created my "first" character since getting into the elysium server, I had created a night elf hunter and I'm finding an issue when it comes to attacking.
I can't right click and auto attack with my bow, it says "you are too far away."
So in short, I have to use the "auto shoot" ability every single time I want to attack something with my bow, its seriously aggravating!!
Am I the only person having this issue??
Anyways, please feel free to get back to me at your earliest convenience as I very much look forward to your response. Thank you very much for taking the time to read this and THANK YOU for your continued service to help bring back the glory days of Vanilla. Cheers!!

-Andrew M.

**Comment by Sabre142** [comment]
Sadly not the same way it works in later versions. To autoshoot you have to either press the specific ability or use any ranged spell. There are certain macros you can also use.

**Comment by Bigmason123** [comment]
Okie doke.
What is the macro for it?

**Comment by Sabre142** [comment]
/script if not IsCurrentAction(60) then UseAction(60) end;
This macro command will click on BottomRight Action Button 12 so place your Attack or Auto Shoot there and you character will start attacking automatically every time you use that macro.

EXAMPLE
/script if not IsCurrentAction(60) then UseAction(60) end;
/cast Arcane Shot
